docs/guides/admin/docs/configuration/basic.md

```diff
@@ -3,106 +3,102 @@ Basic Configuration
-authentication. Therefore anyone who obtains write access to this socket is able to shutdown karaf and everything
-that runs on it. There is a default `karaf.shutdown.command` defined in `custom.properties`. Change this to something
-secret.
+Opencast requires Apache ActiveMQ to relay messages between micro-services.
+For configuration details, please follow the:
 
+- [Apache ActiveMQ configuration guide](message-broker.md)
 
-Step 4: Setting up Apache ActiveMQ Message Broker
--------------------------------------------------
-<!-- _This is a very nonchalant and subdued description for how important this config probably is -->
-Since version 2.0, Opencast requires a running Apache ActiveMQ instance with a specific configuration.  The message
-broker is mostly run on the admin server of Opencast but can be run separately. It needs to be started before Opencast.
-For more details about the setup, have a look at the [Apache ActiveMQ configuration guide](message-broker.md).
 
-
-Step 5: Database Configuration
+Step 4: Database Configuration
 ------------------------------
 
-Opencast uses an integrated H2 database by default. While you will find it perfectly functional, it has certain
-drawbacks:
+Opencast uses an integrated H2 database by default, which has certain drawbacks:
 
-* It is rather slow
 * It cannot be used for distributed set-ups
 * Upgrading Opencast with this database is not possible
 
-For testing, it is totally fine to keep the internal database, but you are highly encouraged to switch to a stand-alone
-database for productional use. For more information about database configuration, have a look at the [Database
-Configuration](database.md) section.
+For testing, it is fine to keep the internal database, but configure a stand-alone database for production.
+Details about the configuration can be found at:
+
+- [Database Configuration](database.md)
+
 
-Step 6: Elasticsearch Configuration
+Step 5: Elasticsearch Configuration
 -----------------------------------
 
-Opencast requires Elasticsearch.
-For more details about the setup, have a look at the [Elasticsearch configuration
-guide](../modules/searchindex/elasticsearch.md).
+Opencast requires Elasticsearch for quick full-text search and data caching.
+For more details about the setup, have a look at the:
 
-Step 7: HTTPS Configuration
+- [Elasticsearch configuration guide](../modules/searchindex/elasticsearch.md).
+
+
+Step 6: HTTPS Configuration
 ---------------------------
-<!-- _Every other instance of linking to another doc so far has been a "do now, but in another doc", 
-whereas here it is very much "not necessary for right now": what to do? -->
-An installation without HTTPS does not make much sense today.
-Thus, make sure to follow [a configuration guide for HTTPS](https/index.md).
 
-Step 8: Setting the Storage Directory (optional)
+To make Opencast available externally and securing connections to it, follow one of the
```

Comment:
`To make Opencast available externally and securing connections to it, ` -> `To secure connections to Opencast`

Reply:
`To make Opencast available externally ` This part should be even more clear, there was an adopter with this trouble because the guide didn't clear that to have external connections, YOU NEED* to configure HTTPS, otherwise, only localhost can access. 

* Yes you can configure `org.ops4j.pax.web.cfg` but is not recommended, I will left this in a troubleshoot guide or something like that.

Reply:
You don't *need* to configure https, it's sane to provide access via http-only.  It's likely not something that most adopters will want to do, but we shouldn't make it sound like https is required when it's not.

Reply:
For the sake of clarity, this guide should provide one way forward, not multiple ways. We should assume that new users do not know what they want (otherwise, they wouldn't read this guide). Providing options like “you can do this but if you don't do it, you need to do something else instead which is something you wouldn't need to do if you did the first thing” just makes the instructions way more confusing and will leave people in a state of “what the hell do I really need to do”.

The first paragraph of this guide states that there are alternatives to what is described in here but this is the recommended set-up. Users should not skip any step except if it is explicitly marked as optional.

I'll try to make this sentence more clear though.